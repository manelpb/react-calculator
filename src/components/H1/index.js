import styled from 'styled-components';

const H1 = styled.h1`
  font-size: 1.5em;
  text-align: center;
  color: red;
`;

export default H1;