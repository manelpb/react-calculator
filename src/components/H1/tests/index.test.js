import React from 'react';
import H1 from '../';
import { shallow } from 'enzyme';

it('should render a h1', () => {
    const render = shallow(<H1 />);
    expect(render.find('h1').length).toEqual(1);
    expect(render).toMatchSnapshot();
})