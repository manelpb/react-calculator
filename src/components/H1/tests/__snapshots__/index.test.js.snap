// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a h1 1`] = `
<h1
  className="sc-bdVaJa eBSsgv"
/>
`;
